packages/firestore/src/api/credentials.ts

```diff
@@ -227,6 +230,9 @@ export class FirebaseCredentialsProvider implements CredentialsProvider {
     this.changeListener = null;
   }
 
+  // Auth.getUid() can return null even with a user logged in. It is because getUid() is synchronous, but
```

Comment:
Do we have a line length recommendation? go/tsstyle used to recommend 80, but I don't see it there anymore, perhaps because all google3 code is auto-formatted now. In any case, these lines seem long.

Reply:
shortened the comment lines.